packages/core/src/plugins/html-deserializer/utils/stripWhitespaceConfig.ts

```diff
@@ -0,0 +1,13 @@
+let stripWhitespace = true;
```

Comment:
Can you pass this down the call stack using options arguments rather than using a global variable?